data/sql/updates/pending_db_world/rev_1605128047204479800.sql

```diff
@@ -0,0 +1,142 @@
+    SELECT `id`,
+            1 AS requirement_type,
+            `quest_done_A`,
+            0 AS faction,
+			quest_failed_text
+    FROM `access_requirement`
+    WHERE `quest_done_A` > 0 AND `quest_done_A` != `quest_done_H`
+    UNION
+    -- Horde quests only
+    SELECT `id`,
+            1 AS requirement_type,
+            `quest_done_H`,
+            1 AS faction,
+			quest_failed_text
+    FROM `access_requirement`
+    WHERE `quest_done_H` > 0 AND `quest_done_A` != `quest_done_H`
+    UNION
+    -- Both factions
+    SELECT `id`,
+            1 AS requirement_type,
+            `quest_done_H`,
+            2 AS faction,
+			quest_failed_text
+    FROM `access_requirement`
+    WHERE `quest_done_H` > 0 AND `quest_done_A` = `quest_done_H`
+;
+
+-- REORDER PRIMARY KEY
+CREATE TEMPORARY TABLE IF NOT EXISTS `temp` select * from `dungeon_access_requirements` ORDER BY dungeon_access_id, faction;
+TRUNCATE `dungeon_access_requirements`;
+ALTER TABLE `dungeon_access_requirements`; 
+INSERT INTO `dungeon_access_requirements` SELECT * FROM `temp`;
+
+-- Verify result with this
+-- SELECT `id`, `quest_failed_text` FROM `access_requirement` WHERE `quest_failed_text` IS NOT NULL;
+
+
+-- Remove columns + rename table
+ALTER TABLE `access_requirement`
+DROP `item`,
+DROP `item2`,
+DROP `quest_done_A`,
+DROP `quest_done_H`,
+DROP `completed_achievement`,
+DROP `quest_failed_text`,
+RENAME TO `dungeon_access_template`,
+COMMENT='Dungeon access template and single requirements';
+
+-- Rename columns
+ALTER TABLE `dungeon_access_template`
+CHANGE `mapId` `instance_id` mediumint unsigned NOT NULL COMMENT 'Map ID from instance_template' AFTER `id`,
+CHANGE `difficulty` `difficulty` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '5 man: 0 = normal, 1 = heroic, 2 = epic (not implemented) | 10 man: 0 = normal, 2 = heroic | 25 man: 1 = normal, 3 = heroic' AFTER `instance_id`,
+CHANGE `level_min` `min_level` tinyint unsigned NOT NULL DEFAULT 0 AFTER `difficulty`,
+CHANGE `level_max` `max_level` tinyint unsigned NOT NULL DEFAULT 0 AFTER `min_level`,
+CHANGE `item_level` `min_avg_item_level` smallint unsigned NOT NULL DEFAULT 0 COMMENT 'Min average ilvl required to enter' AFTER `max_level`,
+CHANGE `comment` `comment` varchar(255) COLLATE 'utf8_general_ci' NULL COMMENT 'Dungeon Name 5/10/25/40 man - Normal/Heroic' AFTER `min_avg_item_level`;
+
+-- Add KEY CONSTRAINTS
+ALTER TABLE `dungeon_access_template` ADD CONSTRAINT `FK_dungeon_access_template__instance_template` FOREIGN KEY (`instance_id`) REFERENCES `instance_template` (`map`) ON DELETE CASCADE ON UPDATE CASCADE;
+ALTER TABLE `dungeon_access_requirements` ADD CONSTRAINT `FK_dungeon_access_requirements__dungeon_access_template` FOREIGN KEY (`dungeon_access_id`) REFERENCES `dungeon_access_template` (`id`) ON DELETE CASCADE ON UPDATE CASCADE;
+
+
+-- Add the acore_strings
+
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES (882, 'To enter, you must complete the following quest(s):');
```

Comment:
id already taken

Reply:
I think it was not taken on my fresh db.

Reply:
yes, my bad